java/core/src/main/java/co/worklytics/psoxy/rules/generics/Calendar.java

```diff
@@ -42,16 +46,20 @@ public class Calendar {
         "Prepare "
     );
 
-    public static final String PREP_TIME_BLOCK_SNIPPETS_PATTERN = "(?i)(" +
-        PREP_TIME_BLOCK_TITLE_SNIPPETS.stream()
-            .map(s -> Pattern.quote(s))
-            .collect(Collectors.joining("|"))
-        + ")";
+    public static final List<String> OOO_TITLE_SNIPPETS = Arrays.asList(
+        "OOO",
+        "OOO:",
+        "Out of Office",
+        "Out of Office:",
+        "Out of the Office",
+        "Out of the Office:"
```

Comment:
Worth to add?.. 

```
O-O-O
Out-of-Office
O-O-O:
Out-of-Office:
```

?

Reply:
that's not in tokens we currently deal with for Calendar 3 methodology.  I don't think those cases common enough to deal with.

(most calendar tools have native OOO fields anyways, so the parsing out of calendar titles imho is likely to be pretty incremental; but might as well do it for now bc I do think historically is still relevant)